ui/admin/app/components/composed/ComboBox.tsx

```diff
@@ -25,22 +25,25 @@ type BaseOption = {
 
 type GroupedOption<T extends BaseOption> = {
     heading: string;
-    value: T[];
+    value: (T | GroupedOption<T>)[];
 };
 
 type ComboBoxProps<T extends BaseOption> = {
     allowClear?: boolean;
     clearLabel?: ReactNode;
+    emptyLabel?: ReactNode;
     onChange: (option: T | null) => void;
-    options: T[] | GroupedOption<T>[];
+    options: (T | GroupedOption<T>)[];
     placeholder?: string;
+    suggested?: string[];
```

Comment:
suggested feels like an implementation detail that should be handled by the parent. Could we instead expose a `renderItem` prop that allows the parent to implement each row as it pleases?